preprocessor.js

```diff
@@ -10,19 +11,9 @@ module.exports = {
           fileName: path
         });
 
-      const modified = `require('ts-jest').install({environment: 'node', emptyCacheBetweenOperations: true});${transpiled.outputText}`;
-
-      return modified;
+      return transpiled.outputText;
```

Comment:
After some checks I've figured out that those things do nothing at production but ruins tests.